message_app_server/message_app/__init__.py

```diff
@@ -27,6 +28,7 @@ def create_app(test_config=None, name=__name__):
         # Disable tracking modification of objects which requires extra memory
         SQLALCHEMY_TRACK_MODIFICATIONS=False
     )
+    CORS(app)
```

Comment:
Using CORS like this would allow CORS for all domains on all routes, which is something we NEVER want. We only accept requests from the domain of the React client app.

Reply:
@vuongtranluc You can change the configuration to `CORS(app, origins=["http://localhost:3000"])` to allow the react app to send requests to the server, but before production, we should have a meeting about which domains are allowed to send requests.